docs/dyndns/update-api.rst

```diff
@@ -206,12 +199,34 @@ Basic authentication with forced use of IPv6 (while preserving the IPv4 address
   curl "https://update6.dedyn.io/?hostname=<your domain>&myipv4=preserve" \
     --header "Authorization: Token <your token secret>"
 
-Basic authentication with simultaneous update of IPv4 and IPv6, option 1::
-
-  curl --user <your domain>:<your token secret> \
-    "https://update.dedyn.io/?myipv4=1.2.3.4&myipv6=fd08::1234"
-
-or option 2::
+Basic authentication with simultaneous update of IPv4 and IPv6::
 
   curl "https://update.dedyn.io/?hostname=<your domain>&myipv4=1.2.3.4&myipv6=fd08::1234" \
     --header "Authorization: Token <your token secret>"
+
+**Examples without headers** (not recommended)
+
+Basic authentication with automatic IP detection (IPv4 **or** IPv6)::
+
+curl --user <your domain>:<your token secret> https://update.dedyn.io/
+
+Basic authentication with forced use of IPv4 (will remove IPv6 address from the DNS)::
+
+curl --ipv4 --user <your domain>:<your token secret> https://update.dedyn.io/
+
+Basic authentication with forced use of IPv4 (while preserving the IPv6 address from the DNS)::
+
+curl --ipv4 --user <your domain>:<your token secret> "https://update.dedyn.io/&myipv6=preserve"
```

Comment:
Quoting from https://github.com/desec-io/desec-stack/pull/956#pullrequestreview-2303327448:
> - Please make sure that all examples work
> - Please make sure that all examples are technically correct

Have you tested this?

The first query parameter should be indicated with `?`, not with `&`. (The quotes can then also go away, they are also needed because Linux command lines have special treatment for `&`.)